config/config-example.js

```diff
@@ -400,6 +400,12 @@ exports.grouplist = [
 		minigame: true,
 	},
 	{
+		symbol: '^',
+		id: "retired",
+		name: "Retired",
```

Comment:
This means that on the pop up it'll say Global Retired. I feel like `Retired Auth` Would look a LOT nicer. The id can stay the same though